licenses.d2iq.yaml

```diff
@@ -74,11 +74,6 @@ resources:
       - license_path: LICENSE
         ref: v${image_tag}
         url: https://github.com/fluent/fluent-bit
-  - container_image: docker.io/aquasec/kube-bench:v0.6.10
```

Comment:
This is venturing into a realm I don't quite understand - but it seems like kube-bench is still getting shipped with DKP - and this may be sorted out some other time, but should this get removed from the licenses?

Reply:
so, I'm not 100% sure of how this is handled but I'm sure that the up to date bundles contain all images used in Insights including kube-bench.
the licenses build was failing with this container_image entry fro kube-bench and so I deleted it manually